src/main/java/microsoft/exchange/webservices/data/search/filter/SearchFilter.java

```diff
@@ -1085,17 +1085,6 @@ protected void internalValidate() throws ServiceValidationException {
       if (this.otherPropertyDefinition == null && this.value == null) {
         throw new ServiceValidationException(
             "Either the OtherPropertyDefinition or the Value property must be set.");
-      } else if (value != null) {
```

Comment:
The next line was:
```
if (!((value instanceof Object) || (value instanceof ISearchStringProvider)))
```

Any object is instance of Object class, so this expressions is always "true", it means that full expressions is always "false".